core/templates/dev/head/pages/exploration_editor/editor_tab/customize_interaction_modal_directive.html

```diff
@@ -29,12 +29,18 @@ <h3 ng-if="stateInteractionIdService.displayed && (customizationModalReopened ||
 
   <form ng-if="stateInteractionIdService.displayed && hasCustomizationArgs" name="form.schemaForm" class="protractor-test-interaction-editor">
     <div ng-repeat="customizationArgSpec in customizationArgSpecs track by $index">
-      <div class="oppia-interaction-customization-label"><[customizationArgSpec.description]></div>
-      <div>
+      <div ng-if="customizationArgSpec.schema.type!='bool'" class="oppia-interaction-customization-label"><[customizationArgSpec.description]></div>
```

Comment:
- Wrap both this and the next div into a single div that has the ng-if.
- Use !== instead of !=.
- Put spaces on the left and right of the !==.